owncloudDomain/src/main/java/com/owncloud/android/domain/mappers/RemoteMapper.kt

```diff
@@ -0,0 +1,27 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author David González Verdugo
+ * Copyright (C) 2019 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.domain.mappers
+
+interface RemoteMapper<Model, Remote> {
+
+    fun toModel(remote: Remote?): Model?
+
+    fun toRemote(model: Model?): Remote?
```

Comment:
Do we need a toRemote? Because it is not implemented anywhere

Reply:
Maybe we don't need it now but a mapper usually map in both directions and we could need it in the future.